src/dodal/devices/stages.py

```diff
@@ -0,0 +1,36 @@
+from ophyd_async.core import Device
+from ophyd_async.epics.motion.motor import Motor
+
+
+class ThreeAxisStage(Device):
```

Comment:
Should: Can we combine this with [XYZPositioner](https://github.com/DiamondLightSource/dodal/blob/00c5c72031e1e8948b59c61ec54b95919fbfae7a/src/dodal/devices/motors.py#L10)